x-pack/plugins/cases/public/common/components/links/index.tsx

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
I'm not sure we should have a `public/common/components` directory, but instead put all components in `public/components`. In security_solutions we have so many different apps working off of `common/components`. But in cases we are only exporting Cases ui components. What do you think @XavierM ?

Reply:
I agree with you but I will like to make sure I am not missing something from @michaelolo24

Reply:
Yep, totally agreed it should all be together. Just tried to maintain as much similarity to the current structure in security solution and then handle refactoring after everything was moved over.